hack/bin/cabal-run-tests.sh

```diff
@@ -14,8 +14,8 @@ fi
 for cabal in $(find "$TOP_LEVEL" -name "$pattern" | grep -v dist-newstyle); do
   # This is required because some tests (e.g. golden tests) must be run from
   # the package root.
-  cd "$(dirname $cabal)"
-  package="$(basename ${cabal%.*})"
+  cd "$(dirname "$cabal")"
+  package="$(basename "${cabal%.*}")"
```

Comment:
this looks wrong to me. too many quotes.

Reply:
a bit intuitive having quotes inside quotes, but it is correct https://stackoverflow.com/questions/46376773/bash-is-subshell-output-implicitly-quoted